### ISSUE TITLE
Drag and drop files

### ISSUE THREAD

**Issue by GNHua** [mdn/dom-examples#3]
The `ondragend` does not fire in `File-drag.html`.

**Comment by ollelauribostrom** [comment]
Also noticed this when running the examples. According to the note on the [HTML Drag and Drop API page](https://developer.mozilla.org/en-US/docs/Web/API/HTML_Drag_and_Drop_API) dragstart and dragend events are not fired when dragging a file into the browser from the OS. Maybe the example and the associated text could be updated (I'd love to help). Is there any reason why the cleanup can’t happen after the files has been processed in the drop handler?

**Comment by chrisdavidmills** [comment]
@ollelauribostrom Yes, this does like it could use some cleanup. I've been so busy that I've just not had the time to look at this. If you did have a little time update the example and text, I'd be very grateful ;-)

**Comment by ollelauribostrom** [comment]
@chrisdavidmills Opened a PR and did some changes to [the text](https://developer.mozilla.org/en-US/docs/Web/API/HTML_Drag_and_Drop_API/File_drag_and_drop). Let me know what you think!
